wrappers/javascript/packages/anoncreds-nodejs/src/NodeJSAnoncreds.ts

```diff
@@ -617,6 +627,380 @@ export class NodeJSAnoncreds implements Anoncreds {
     return new ObjectHandle(handleReturnPointer<number>(ret))
   }
 
+  public createW3CCredentialOffer(options: {
+    schemaId: string
+    credentialDefinitionId: string
+    keyCorrectnessProof: ObjectHandle
+  }): ObjectHandle {
+    const { schemaId, credentialDefinitionId, keyCorrectnessProof } = serializeArguments(options)
+
+    const ret = allocatePointer()
+    this.nativeAnoncreds.anoncreds_create_w3c_credential_offer(
```

Comment:
As I mentioned in another comment before, I do't think we need a w3c credential offer method, as the offer is not any different based on whether it's a w3c or anoncreds offer? 

The only difference is the end-result: whether the credential is encoded as AnonCreds or W3C credential